packages/tasks/src/model-libraries.ts

```diff
@@ -100,7 +96,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "audioseal",
 		repoUrl: "https://github.com/facebookresearch/audioseal",
 		filter: false,
-		countDownloads: { wildcard: { path: "*.pth" } },
+		countDownloads: "path_extension:pt",
```

Comment:
I think it's `pth`, can you confirm @Vaibhavs10?

Reply:
ah yes forgot one character